source/deployment/mobile-app-deployment.rst

```diff
@@ -204,33 +190,27 @@ For those taking on larger deployments, we assume you are already using an EMM p
 
 AppConfig is newer, more modern approach when compared to the previous standard, App Wrapping.
 
-We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
-
-While app wrapping has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
-
-AppConfig (Supported) vs. App Wrapping (Not Supported)
+AppConfig (supported) vs. app wrapping (not supported)
 ======================================================
 
     **Note:** Mattermost only supports the AppConfig standard. It does not support app wrapping. Use app wrapping at your own risks.
 
-So app wrapping isn’t supported, and the AppConfig standard is, but what is the difference between the two?
-
-While there are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining both, let’s take a look at AppConfig first.
+There are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining AppConfig and app wrapping, let us take a look at AppConfig first.
```

Comment:
Should we change this to "a great article" or possibly include a few more articles to round it off?

Reply:
I'd change it to 

> Here is a great article ...

WDYT @justinegeffen

Reply:
I'd possibly reword it to "Here is a helpful article" as opposed to "great" unless we're certain that the referenced article is considered great by multiple trusted sources. 3/5 as this type of writing is different to technical documentation.

Reply:
I'm 0/5 here, I just went with the wording in your suggestion ;)